pkg/ingester/otlp/convert.go

```diff
@@ -30,6 +30,15 @@ func ConvertOtelToGoogle(src *otelProfile.Profile) *googleProfile.Profile {
 		Comment:           src.Comment,
 		Mapping:           convertMappingsBack(src.Mapping),
 	}
+	stringmap := make(map[string]int)
+	addstr := func(s string) int64 {
+		if _, ok := stringmap[s]; !ok {
+			stringmap[s] = len(dst.StringTable)
+			dst.StringTable = append(dst.StringTable, s)
+		}
+		return int64(stringmap[s])
+	}
+	addstr("")
```

Comment:
Is this needed?

Reply:
I think it is better to do it explicitly for https://github.com/google/pprof/blob/1e3fdc7de4679692c8c3bafdb87274f45090c73a/proto/profile.proto#L66